src/main/java/linenux/model/Schedule.java

```diff
@@ -41,6 +42,27 @@ public void deleteTask(Task task) {
     }
 
     /**
+    * View the specific task.
```

Comment:
Formatting should not be done in the model class ! Retrieve the task from the schedule and format in your view command